empress/fileInput.py

```diff
@@ -0,0 +1,220 @@
+# ReconInput class
+from io import StringIO
+
+# BioPython libraries
+from Bio import Phylo
+
+class ReconInput(object):
+    """
+    Storage class for the newick data (trees, tip mapping, and optional distance parameters)
+    Distances encode branch lengths in the newick file, if given
+    """
+
+    def __init__(self, host_tree = None, host_distances = None, parasite_tree = None, 
+                parasite_distances = None, phi = None):
+        self.host_tree = host_tree
```

Comment:
After you write the function that verifies whether the tree and the mapping is correct, it would be nice if you add the verification here too. :)